code/src/components/GameScreen.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import styled from 'styled-components';
+
+import { continueGame } from 'reducers/game';
+
+const GameScreen = () => {
+	const dispatch = useDispatch();
+
+	const actions = useSelector((store) => store.game.position.actions);
+	const setDescription = useSelector(
+		(store) => store.game.position.description
+	);
+
+	const onContinueGame = (type, direction) => {
+		dispatch(continueGame(type, direction));
+	};
+
+	return (
+		<GameContainer>
+			{setDescription && (
+				<StyleMainDescription className='nes-balloon from-left nes-pointer'>
+					{setDescription}
+				</StyleMainDescription>
+			)}
+			{actions &&
```

Comment:
This works super well, if you wanted to add a restart button I think it would be fairly easy to put in as maybe a ternary operator for when actions is empty, and in that case also render a restart button! :)